build_projects/update-dependencies/update-dependencies.ps1

```diff
@@ -34,12 +42,17 @@ if($LASTEXITCODE -ne 0) { throw "Failed to install stage0" }
 # Put the stage0 on the path
 $env:PATH = "$env:DOTNET_INSTALL_DIR;$env:PATH"
 
+# Generate some props files that are imported by update-dependencies
```

Comment:
I don't think update-dependencies actually needs to import the dirs.props in the root.  Can we just remove the Import in its .csproj instead?  That way this call isn't duplicated.